src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -14,6 +14,7 @@
             [liberator.core :refer [defresource]]
             ;[liberator.dev :as liberator-dev]
```

Comment:
I left them in there because I thought they would be useful to people in the future.  They can be extremely useful in debugging issues with liberator resources, and I figured most people didn't know about the time (at the time, anyway).

If y'all don't think they're useful now, I am fine if they get removed.